packages/language-server/src/diagnosticsHandler.ts

```diff
@@ -0,0 +1,34 @@
+import { Diagnostic, DiagnosticSeverity, DiagnosticTag } from 'vscode-languageserver/node'
+import { getBlockAtPosition } from './util'
+
+export function greyOutIgnoredParts(lines: string[]): Diagnostic[] {
+  const diagnostics: Diagnostic[] = []
+
+  lines.map((currElement, index) => {
+    if (currElement.includes('@@ignore')) {
+      const block = getBlockAtPosition(index, lines)
+      if (block) {
+        diagnostics.push({
+          range: { start: block.range.start, end: block.range.end },
+          message:
+            '@@ignore: this model will be kept in sync with the database schema, however, it won’t be exposed to the Prisma client.',
+          tags: [DiagnosticTag.Unnecessary],
```

Comment:
This is what dims out the given section!

Reply:
Oh I see! It looks like it was added in the Language Server Protocol version 3.15.0:
```
/**
	 * Unused or unnecessary code.
	 *
	 * Clients are allowed to render diagnostics with this tag faded out
	 * instead of having an error squiggle.
	 */
```
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnostic
```
		/**
		 * Unused or unnecessary code.
		 *
		 * Diagnostics with this tag are rendered faded out. The amount of fading
		 * is controlled by the `"editorUnnecessaryCode.opacity"` theme color. For
		 * example, `"editorUnnecessaryCode.opacity": "#000000c0" will render the
		 * code with 75% opacity. For high contrast themes, use the
		 * `"editorUnnecessaryCode.border"` the color to underline unnecessary code
		 * instead of fading it out.
		 */
```
https://github.com/Microsoft/vscode/blob/83625fa9cbbdec3eab4a635c62f88aab0929aa80/src/vs/vscode.d.ts#L3945

Reply:
LSP v3.15.0 is available since https://github.com/Microsoft/vscode-languageserver-node version 6.0.0
Looks good!